themes/default/content/blog/pulumi-release-notes-0.57/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Pulumi release notes: Milestone 58"
+date: 2021-06-08T11:47:31-07:00
+draft: false
+meta_desc: "New in Pulumi: Amazon ECS Anywhere and App Runner, Sumo Logic support, Pulumi Packages updates, and easier updates for teammates in the Pulumi Service"
+meta_image: meta.png
+authors:
+    - alex-mullans
+tags:
+    - features
+---
+
+We release new versions of the Pulumi CLI every two weeks, new versions of the Pulumi Service multiple times a week, and new and updated cloud providers frequently. To keep up with all of these changes, you used to have to find the `CHANGELOG.md` files in the repo (or repos!) that you care about, which could get unwieldy. So, we're trying something new: a tri-weekly rollup of Pulumi news, features, and updates. If you find it useful, or have ideas on how we could improve it, let us know [on Twitter](https://twitter.com/pulumi) or in the [Pulumi Community Slack](https://slack.pulumi.com)!
+
+In this update:
+
+- New and updated cloud providers
+  - Pulumi supports Amazon ECS Anywhere and Amazon App Runner
+  - Pulumi supports Sumo Logic
+  - 100 new resources in the Azure Native provider
+- Pulumi engine and core technologies
+  - Pulumi Packages schema documentation now available
+  - Pulumi Packages schemas can reference other packages' types
+  - Input parameters can now be marked as secret
+  - Plugin install output now goes to `stderr`
+  - Additional safety checks during `pulumi import`
+- Pulumi Service and Pulumi.com
+  - Easier acceptance of invites to organizations
+  - Faster resource docs
+  - Self-Hosted Pulumi Service now supports Azure Storage and Minio backends
+
+<!--more-->
+
+## New and updated cloud providers
+
+### Pulumi supports Amazon ECS Anywhere and Amazon App Runner
+
+Earlier this month, Amazon announced a [new version of their Elastic Container Service](https://aws.amazon.com/blogs/containers/introducing-amazon-ecs-anywhere/) that allows you to deploy containers to almost any server and a new fully-managed container-native service called [AWS App Runner](https://aws.amazon.com/blogs/containers/introducing-aws-app-runner/). Pulumi had full same-day support for both new services.
+
+Learn how to get started with [ECS Anywhere]({{< relref "/blog/ecs-anywhere-launch" >}}) and [App Runner]({{< relref "/blog/deploy-applications-with-aws-app-runner" >}}).
+
+### Pulumi supports Sumo Logic
+
+This milestone, we released a new provider for Sumo Logic. You can now manage Sumo Logic's big data and analytics services with Pulumi infrastructure-as-code.
```

Comment:
```suggestion
The new Sumo Logic provider for Pulumi allows you to manage Sumo Logic's big data and analytics services with Pulumi infrastructure-as-code.
```

Perhaps include the code example (with lang chooser) from https://www.pulumi.com/docs/intro/cloud-providers/sumologic/ inline here to liven this up a little (have some code)?